test/view_models/notifications_viewmodel_test.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:hng/ui/view/notifications/notifications_viewmodel.dart';
+import '../helpers/test_helpers.dart';
+
+void main() {
+  group('notificationsViewModelTest -', () {
+    setUp(() => registerServices());
+    tearDown(() => unregisterServices());
+
+    group('initialise -', () {
+      test('When constructed, checks if all the fields have a default state of true', () {
+        final model = NotificationsViewModel();
```

Comment:
This particular test is invalid, since these values will be gotten from the local storage after being retrieved from the server